src/main/kotlin/com/example/lottery/lotteries/domain/LottoDomain.kt

```diff
@@ -0,0 +1,86 @@
+package com.example.lottery.lotteries.domain
+
+import java.time.*
+import java.time.temporal.ChronoUnit
+import kotlin.random.Random
+
+object LottoDomain {
+
+    // 로또 번호 결과 데이터 클래스
+    data class LottoNumbersResult(val numbers: List<Int>)
+
+    // 로또 순위 결과 데이터 클래스
+    data class RankingResult private constructor(
+        val isWin: Boolean,
+        val place: Int,
+        val rewardType: RewardType,
+        val rewardAmount: Int,
+    ) {
+        companion object {
+            private const val NO_WIN_PLACE = -1 // 미당첨시 순위
+
+            fun lose() = of(NO_WIN_PLACE)
+
+            fun of(place: Int): RankingResult {
+                val (rewardType, amount) = when (place) {
+                    1 -> RewardType.NONE to 10_000_000
+                    2 -> RewardType.NONE to 1_000_000
+                    3 -> RewardType.POINT to 50_000
+                    4 -> RewardType.POINT to 1_000
+                    5 -> RewardType.LOTTERY_COIN to 1_000
+                    else -> RewardType.NONE to 0
+                }
+                return RankingResult(place != NO_WIN_PLACE, place, rewardType, amount)
+            }
+        }
+    }
+
+    enum class RewardType {
+        NONE,
+        POINT,
+        LOTTERY_COIN,
+    }
+
+    // 로또 번호 생성 로직
+    fun generateLottoNumbers(): LottoNumbersResult {
+        val numbers = (1..45).toMutableList()
+        val lottoNumbers = mutableListOf<Int>()
+        repeat(6) {
+            val randomIndex = Random.nextInt(numbers.size)
+            lottoNumbers.add(numbers.removeAt(randomIndex))
+        }
+        return LottoNumbersResult(lottoNumbers.sorted())
+    }
+
+    // 로또 회차 계산 로직
+    fun getCurrentLottoRound(): Int {
+        val firstLottoDate = LocalDate.of(2002, 12, 7)
+        val zoneId = ZoneId.of("Asia/Seoul")
+        val now = ZonedDateTime.ofInstant(Instant.now(), zoneId)
+        val todayLottoTime = ZonedDateTime.of(now.toLocalDate(), LocalTime.of(21, 0, 0), zoneId)
+        val effectiveNow = if (now.isBefore(todayLottoTime)) now.minusDays(1) else now
+        val weeksBetween = ChronoUnit.WEEKS.between(firstLottoDate, effectiveNow.toLocalDate())
+        return (weeksBetween + 1).toInt()
+    }
```

Comment:
```suggestion
    fun getCurrentLottoRound(): Int {
        val start = ZonedDateTime.parse("2002-12-07T21:00:00+09:00")
        val now = Instant.now().atZone(ZoneId.of("Asia/Seoul"))
        val weeksBetween = ChronoUnit.WEEKS.between(start, now)        
        return (weeksBetween + 2).toInt()
    }
```
이렇게 표현하면 더 간결할 수 있지 않을까 생각해봤어요